packages/shared/src/composables/use-items.ts

```diff
@@ -205,4 +210,38 @@ export function useItems(collection: Ref<string | null>, query: ComputedQuery, f
 		const endpoint = computed(() => `/utils/sort/${collection.value}`);
 		await api.post(endpoint.value, { item, to });
```

Comment:
Unrelated observation here, but maybe we can remove the usage of computed here to make it cleaner?

or if we want to be safer, maybe we can mirror the same return null logic when `collection.value` is not set:

https://github.com/directus/directus/blob/8f5a0723525516f8e9e1c26515f64446b146748f/packages/shared/src/composables/use-items.ts#L41-L46

and prevent this POST request from firing, similar to getItems' approach:

https://github.com/directus/directus/blob/8f5a0723525516f8e9e1c26515f64446b146748f/packages/shared/src/composables/use-items.ts#L103

Reply:
Gonna compile this as a larger refactoring exercise 👍🏻